x-pack/plugins/security_solution/public/exceptions/translations/shared_list.ts

```diff
@@ -267,6 +267,13 @@ export const IMPORT_EXCEPTION_LIST_AS_NEW_LIST = i18n.translate(
   }
 );
 
+export const IMPORT_EXCEPTION_ENDPOINT_LIST_WARNING = i18n.translate(
+  'xpack.securitySolution.exceptionsTable.importExceptionEndpointListWarning',
+  {
+    defaultMessage: 'We only allow one Exception List for Endpoint Security.',
```

Comment:
This text maybe needs a review from the docs team. I think this needs to be rephrased to adhere to writing guidelines

Reply:
@caitlinbetz @nastasha-solomon any suggestions on how to adjust this message?

Reply:
Hey, thanks for the ping! Do you have any objections to removing the pronoun (we) in the message? Our current [guidelines](https://eui.elastic.co/#/guidelines/writing/guidelines#addressing-the-user) indicate that "we" is more appropriate in situations when we're taking an action for the user (i.e., helping them) or making a suggestion. In this situation, we're not helping users along. Instead, we're blocking them from doing what they want to do and then telling them that _we_ are the ones blocking them. If you're ok with removing the pronoun, here are a few options to consider:

- Multiple exception lists for Endpoint Security are not allowed.
- Only one Endpoint Security Exception list is allowed.
- Additional exception lists for Endpoint Security are not supported.

Let me know if these work, or if they're not quite what you're looking for.

Reply:
This looks great! I will go with the first option.